app/models/BasicPokemonDetail.js

```diff
@@ -9,6 +9,182 @@ let BasicPokemonDetailSchema = new mongoose.Schema({
     },
     name: {
         type: String
+    },
+    gender: {
+        /*male: {
```

Comment:
Well, if I say gender = 1,2 or, more concretely, gender: male = true, false, is a kind of gustus.